### PR TITLE
mv ngql-native

### PR DESCRIPTION
**Pull request by whitewum**

### CODE REVIEW

**Review by cooper-lzy — DISMISSED** [review]

**Review by FengUyu — APPROVED** [review]

File: docs-2.0/19.FAQ/0.FAQ.md [hunk 1]
```diff
@@ -171,15 +86,15 @@ Storage服务使用Raft协议（多数表决），为保证可用性，要求出
 
 ### 如何处理错误信息`can’t solve the start vids from the sentence`
 
-查询引擎需要知道从哪些VID开始图遍历. 这些开始（图遍历）的VID，或者通过用户指定，例如
+查询引擎需要知道从哪些VID开始图遍历。 这些开始图遍历的VID，或者通过用户指定，例如
```
Comment (randomJoe211) [inline docs-2.0/19.FAQ/0.FAQ.md:89]: Extra space detected between "。" and "这".

Lack of punctuation detected after "例如".

File: docs-2.0/19.FAQ/0.FAQ.md [hunk 2]
```diff
@@ -171,15 +86,15 @@ Storage服务使用Raft协议（多数表决），为保证可用性，要求出
...
 
 ```ngql
 > GO FROM ${vids} ...
 > MATCH (src) WHERE id(src) == ${vids}
 # 开始图遍历的VID通过如上办法指定
 ```
 
-或者通过一个(属性)索引来得到，例如
+或者通过一个属性索引来得到，例如
```
Comment (randomJoe211) [inline docs-2.0/19.FAQ/0.FAQ.md:97]: Lack of punctuation detected after "例如".

File: docs-2.0/3.ngql-guide/1.nGQL-overview/1.overview.md [hunk 3]
```diff
@@ -67,3 +67,95 @@ CREATE {TAG | EDGE} {<tag_name> | <edge_type>}(<property_name> <data_type>
 ```ngql
 nebula> CREATE TAG player(name string, age int);
 ```
+
+## 关于openCypher兼容性
+
+## 原生 nGQL 和 openCypher 的关系
+
+原生 nGQL 是由 Nebula Graph 自行创造和实现的图查询语言。OpenCypher 是由 openCypher Implementers Group 组织所开源和维护的图查询语言，
+最新版本为 openCypher 9。由于 nGQL 语言部分兼容了 openCypher，这个部分在本文中称为 OpenCypher 兼容语句。
```
Comment (randomJoe211) [inline docs-2.0/3.ngql-guide/1.nGQL-overview/1.overview.md:76]: The last "OpenCypher" should not be capitalized.

Comment (whitewum) [reply docs-2.0/3.ngql-guide/1.nGQL-overview/1.overview.md:76]: let's ignore the minor just for now.
And proofread later.

**Review by cooper-lzy — APPROVED** [review]

**Review by FengUyu — APPROVED** [review]

**Pull request merged**